frontend/packages/shared/CHANGELOG.MD

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Released]
 
+## [1.6.1] - 2023-02-10
```

Comment:
Tällä hetkellä jo olemassa tällä versiolla paketti, määritelty avoimessa PR:ssä https://github.com/Opetushallitus/kieli-ja-kaantajatutkinnot/pull/421. Laitetaanko vaikka 1.7.0?

Reply:
Tää on tällä hetkellä ehkä vähän huono että ei-mergatuista pullareista tehdään valmiita versioita tuonne pakkaustenhallintaan.

Voin laittaa tähän vaikka just tuon 1.7.0 version jos mergetään sillä, mutta tätä prosessia vois kyllä miettiä uusiksi jos se aiemmin käytetty tapa että deployataan jollain eri nimellä ennen mergausta ja mergen yhteydessä varsinainen versio deployataan pakkaustenhallintaan on hankala.

Reply:
Joo, voidaan toki miettiä prosessia uusiksi jos siltä tuntuu. Nostanpa kuitenkin esille että tästäkin PR:stä olisi varmaan hyvä olla se paketti tuolla repossa katselmointia varten..

Reply:
Mä deployasin nyt tuon 1.7.0:n. Mut tosiaan, mietin tätä myös sitä kautta että jos tuonne nyt pullarin yhteydessä esim. vie "valmiin" version ja muuttaa sharedin koodia jälkeenpäin deployamatta sitä uudelleen ja sit mergetään deviin niin meille päätyy rikkinäinen valmiin version numerolla oleva pakkaus pakkaustenhallintaan.

Yks ratkaisumahdollisuus olis kai myös dev-buildissa tuhota vanha versio samalla versionumerolla jos sellainen jo löytyy?